main.go

```diff
@@ -229,12 +235,52 @@ func visit(url *url.URL) {
 				// 30x but no Location to follow, give up.
 				return
 			}
-			log.Fatal("unable to follow redirect: %v", err)
+			log.Fatalf("unable to follow redirect: %v", err)
 		}
 		visit(loc)
 	}
 }
 
+func responseBody(response io.ReadCloser) string {
+
+	var path string
+	var body string
+	var err error
+	if showBodyEnv := os.Getenv(HTTPSTAT_SHOW_BODY); showBodyEnv != "" {
+		showBody, err = strconv.ParseBool(showBodyEnv)
+		if err != nil {
+			log.Fatalf("unable to parse HTTPSTAT_SHOW_BODY env: %v", err)
+		}
+	}
+
+	if !showBody {
```

Comment:
always put the positive condition first, eg
```
if showBody {
    // do show bodycode
} else {
   // write to file
}
```
This is because the true condition usually comes with a return from the function, which means you can avoid the `else` entirely. This reduces the complexity of the function.